src/toolchain/export-web.md

```diff
@@ -143,6 +145,12 @@ and open it in a supported browser such as Google Chrome or Microsoft Edge.
 
 [godot-export-templates]: https://docs.godotengine.org/en/stable/tutorials/export/exporting_projects.html#export-menu
 
+## Troubleshooting
+
+1. Make sure both `Extensions Support` and `Thread Support` (in Godot 4.3+) are turned on.
+2. Make sure the webserver you're running supports Cross-Origin Isolation (I used `sfz --coi`).
+3. Make sure your rust library and godot project are named differently, as sometimes `.wasm` files are overwritten.
+4. Make sure you're using the `emcc` version in the guide.
```

Comment:
For `1` (regarding Thread Support), see other comment.

For `2`, it's worth mentioning that the Godot editor itself may be used to serve the web game for testing (there are images of this below), but you can also use its `serve.py` script directly if you want: https://github.com/godotengine/godot/blob/e3213aaef5e0e72b8272e65d989d3d8222be17ca/platform/web/serve.py

In addition, the main reason why support for non-threaded builds was added was precisely to work in environments without Cross-Origin Isolation IIRC, so that may be worth mentioning too.

Otherwise, good advice.